mods/ts/rules/defaults.yaml

```diff
@@ -1136,20 +1137,16 @@
 		Type: Heavy
 	LineBuildNode:
 		Types: wall, gate
-	-Building:
 	-Capturable:
 	-GivesBuildableArea:
 	-MustBeDestroyed:
 	-WithSpriteBody:
 	WithGateSpriteBody:
 		OpenSequence: open
 	Tooltip:
-	Gate:
+	Building:
 		BuildSounds: place2.aud
-		OpeningSound: gateup1.aud
-		ClosingSound: gatedwn1.aud
 		TerrainTypes: Clear, Rough, Road, DirtRoad, Green, Sand, Pavement
```

Comment:
...But note that this one *isn't* inherited (it matches walls, which can be placed on more terrain types – is this correct?)

Reply:
I'm 100% sure Green, Sand and Pavement were valid terrain for *all* buildings in TS.

The only potentially wrong terrain type here is 'Rough', but ^BasicBuilding has that as well. I'll just move this list to ^BasicBuilding for now.